Assets/VIRTUOSO/VRTK/Source/SDK/Unity/SDK_UnityController.cs

```diff
@@ -696,6 +907,14 @@ protected virtual void SetTrackedControllerCaches(bool forceRefresh = false)
                         cachedLeftTracker = cachedLeftController.GetComponent<SDK_UnityControllerTracker>();
                         cachedLeftVelocityEstimator = cachedLeftController.GetComponent<VRTK_VelocityEstimator>();
                         SetControllerButtons(ControllerHand.Left);
+#if UNITY_2019_1_OR_NEWER
+                        List<InputDevice> leftHandDevices = new List<InputDevice>();
+                        InputDevices.GetDevicesAtXRNode(XRNode.LeftHand, leftHandDevices);
+                        if(leftHandDevices.Count > 0)
```

Comment:
wouldn't a leftHandDevices[0] != null work too?

Reply:
No, the list can be empty so the leftHandDevices[0]  call would result in an exception